flink-examples/flink-examples-streaming/src/main/java/org/apache/flink/streaming/examples/statemachine/kafka/KafkaStandaloneGenerator.java

```diff
@@ -90,7 +91,7 @@ public void collect(Event evt) {
 
 		@Override
 		public void close() {
-			producer.close();
+			producer.close(Duration.ofSeconds(0));
```

Comment:
Should we also flush the producer here?

Reply:
I will actually revert this change because it's a standalone generator that's not executing in Flink.